### ISSUE TITLE
Using just one tag for alternative decks

### ISSUE THREAD

**Issue by hungriesthippo** [akrivka/roamsr#15]
Wondering how others feel about this. I don't like having to use both my deck tag and #sr to cards.

**Comment by akrivka** [comment]
You can already add cards to different decks top-level. E.g., you can have
- #english cards
    - hello #sr
         - ahoj
    - bye #sr 
        - nashledanou

I definitely want to add to possibility to have multiple main tags though, live #sr, #ir, #fi

**Comment by hungriesthippo** [comment]
Yeah, I'm referring more to the latter kind of case. My IR/FI stuff tends to be one-off, so the top-level block doesn't help there.
